Content.Server/Nutrition/EntitySystems/FoodSystem.cs

```diff
@@ -138,6 +139,16 @@ private void OnFeedFood(Entity<FoodComponent> entity, ref AfterInteractEvent arg
             return (false, true);
         }
 
+        if (TryComp<ContainerManagerComponent>(food, out var containerManager))
+        {
+            // hardcode container name
+            if (containerManager.Containers.TryGetValue("item", out var container) && container.ContainedEntities.Any())
```

Comment:
I think you can TryComp<ContainerContainer> and loop through all containers and their contents for this check.

Reply:
Some entities has SolutionContainerManager, body parts, actions which collects in Containers, so it's too terrible to check. An entry of container could be Container too contains Container and etc.
If someone adds something more, he also would be able to add check here. Better to say that we would check only by name, because items collects only in `item` container.

Reply:
https://github.com/space-wizards/space-station-14/blob/b20fd4fbd00912b6c94f001d2f3ae15d01f0a23a/Content.Shared/Storage/StorageComponent.cs#L17

what about checking StorageComponent + its ContainerId?

Reply:
You mean to check container entities with StorageComponent?

Reply:
I didn't noticed that `ClothingSlotBase` has `ItemSlots`, it's better to use to check... Sorry, my fault